backend/project_review_coordination/models.py

```diff
@@ -56,4 +60,26 @@ def get_projects_user_can_review(user):
             .prefetch_related("content_item", "recruit_project")
         )
 
-        return cards
+        if user.is_superuser:
+            return cards
+        
```

Comment:
I worry about efficiency, we are hitting the database quite a lot. This view is already really slow.

Here is some pseudocode for a faster version:

```
@lru_cache  # I'm not sure if this will work if we use django model instances as arguments. But we should try not to fetch the same users teams over and over 
def get_user_teams(user):
      return user.teams()

...
        viewable_teams = get all the teams the current user can view before we start looping
        if viewable_teams == []:
              return [] 

        for card in cards:
            assignee = card.assignees.first()
            assignee_teams = get_user_teams(assignee)

            for team in assignee_teams:
                 if team in viewable_teams:
                      permitted_cards.append(card)
                      break

return permitted_cards 
```